egs2/fisher_callhome_spanish/asr1/local/data.sh

```diff
@@ -69,7 +64,17 @@ if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
 
     # concatenate multiple utterances
     local/normalize_trans.sh ${sfisher_transcripts} ${callhome_transcripts}
-    ./utils/combine_data.sh data/dev_all data/dev data/dev2
+
+    utils/combine_data.sh \
+        --extra-files "text.lc.en text.lc.es text.lc.rm.en text.lc.rm.es text.tc.en text.tc.es" \
```

Comment:
For ASR, we do not need those extra files, only `text` is needed, so the base `combine_data.sh` should be fine

Reply:
Even if we want to include the case info (I don't think we have the need? correct me if I'm wrong), we can always have an argument and replace `text` with that at here